pyoverkiz/utils.py

```diff
@@ -0,0 +1,64 @@
+"""Utils for Overkiz client"""
```

Comment:
Not a big fan of `utils` name. This is too generic. I would suggest something like `obfuscate.py` or `mask.py`.

Reply:
Fine by me as well, let's change it to what we decide the function name to be.